pages/book/[isbn].tsx

```diff
@@ -0,0 +1,145 @@
+import { Badge, Box, Button, Flex, Grid, HStack, Spacer, Text, Wrap } from "@chakra-ui/react";
+import Image from 'next/image'
+import { PostCardList } from "@components/CardList";
+import Layout from "@components/Layout";
+import { useBookQuery } from "@lib/hooks/book";
+import type { NextPage } from "next";
+import { useRouter } from "next/router";
+import Link from "next/link";
+import { resolveImageUrl } from "@lib/helpers/frontend/resolve-image-url";
+
+const BookPage: NextPage = () => {
+  const router = useRouter();
+  const { isbn } = router.query;
+  const { data: book } = useBookQuery(isbn);
+  if (!book) {
+    return null;
+  }
+  const { name, googleBook, posts } = book;
+
+  const postsWithBookIncluded = posts.map(post => {
+    return {
+      ...post,
+      book: book,
+    }
+  })
+
+  const courseBadges: React.ReactFragment = (
+    <Wrap justify='space-between'>
+      {book.courses.map((course, i) => (
+        <Link key={i} href={'/course/' + course.id} passHref>
+          <Badge variant='subtle' colorScheme='teal' cursor='pointer'>
+            {course.name}
+          </Badge>
+        </Link>
+      ))}
+    </Wrap>
+  ) 
+
+  const bookInfo: React.ReactNode = (
+    <Grid 
+      width='100%' 
+      templateColumns={{
+        base: "256px 1fr",
+      }}
+      templateRows={{
+        base: '300px',
+      }}
+      templateAreas={{
+        sm: `'image image' 'info info'`, 
+        md: `'image info'`,
+      }}
+      gap={8}
+    >
+      <Box gridArea='image'>
+        <Flex 
+          direction='row' 
+          h='100%' 
+          justifyContent='center' 
+          alignItems='center' 
+        >
+          <Image 
```

Comment:
Since the source images are low res I just centered it in this grey box:
![image](https://user-images.githubusercontent.com/44421339/152901037-23c0ee89-4e11-4404-8bf7-3d416f0b17f9.png)

I have played around with it quite a bit and I'm not really sure what would look better but I am open to suggestions

Reply:
I think the grey box looks good!